packages/cli/src/index.ts

```diff
@@ -16,20 +16,24 @@ program
 
 program
   .command('dev')
-  .description('Start a Novu Dev Studio and a localtunnel')
-  .option('-p, --port <value>', 'Set the local port for the Novu endpoint, defaults to 4000', '4000')
-  .option('-r, --route <value>', 'Set the Novu Endpoint mounted route, defaults to /api/novu', '/api/novu')
-  .option('-o, --origin <value>', 'Set the origin for the Novu endpoint')
-  .option('-sp, --studio-port <value>', 'Set the local port for the Novu Local Studio server, defaults to 2022', '2022')
-  .option(
-    '-so, --studio-remote-origin <value>',
-    'Set the remote origin for Novu Studio, used for staging environment and local development, defaults to https://web.novu.co',
-    'https://web.novu.co'
-  )
-  .option(
-    '--region <value>',
-    'Studio Origin SPA, used for staging environment and local development, defaults to us',
-    'us'
+  .description(
+    `Start Novu Studio and a localtunnel
+
+  Running the Bridge application on port 4000: 
+  (e.g., novu dev -p 4000)
+
+  Running the Bridge application on a different route: 
+  (e.g., novu dev -r /v1/api/novu)
+
+  Using Europe region:
+  (e.g., novu dev -cr eu)`
   )
+  .usage('[-p <port>] [-r <route>] [-o <origin>] [-cr <region>] [-cd <dashboard-url>] [-sp <studio-port>]')
+  .option('-p, --port <value>', 'The local Bridge endpoint port', '4000')
+  .option('-r, --route <value>', 'The Bridge endpoint route', '/api/novu')
+  .option('-o, --origin <value>', 'The Bridge endpoint origin')
+  .addOption(new Option('-cr, --region <value>', 'The Cloud Dashboard region').default('us').choices(['us', 'eu']))
+  .option('-cd, --dashboard-url <value>', 'The Cloud Dashboard URL', 'https://dashboard.novu.co')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ why not just `-d`?

Reply:
It was to align with `--cr` in `--region` (which also pertains to a Cloud parameter), for which region had a collision with the `-r` in `--route`. But we are removing region and `-cr` now, so we'll make it `-d` per the suggestion.